plugin/command_history/command_history.py

```diff
@@ -82,11 +82,3 @@ def history_get(number: int) -> str:
         """returns the history entry at the specified index"""
         num = (0 - number) - 1
         return history[num]
-
-    def history_transform_phrase_text(words: list[str]) -> Optional[str]:
```

Comment:
This was available to be context redefined. We should probably have a discussion on the community meetup if we want to keep it as an action.

Reply:
I put the action back.